ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -865,6 +865,28 @@ public void shouldSetShowDescriptionsForShowQueriesDescriptions() {
     assertThat(listQueries.getShowExtended(), is(true));
   }
 
+  @Test
```

Comment:
shall we add negative tests too? (make sure `getShowDescription` is false when it should be)